docs/historias_usuario.md

```diff
@@ -0,0 +1,16 @@
+# Historias de usuario
+[[HU1]](https://github.com/dmonjasm/RouteCheck/issues/2) [Alfonso](https://github.com/dmonjasm/RouteCheck/blob/Objetivo-1/docs/personas.md#alfonso-manso-hernando) utiliza el servicio
+diariamente para llegar a su puesto de trabajo y **ahorrar el máximo dinero posible**. Todas las mañanas comprueba desde el móvil la ruta a seguir. Debe introducir la información
+del vehículo, el precio de la gasolina del sitio donde suela recargar y el origen y destino deseados.
```

Comment:
Lo ideal es que no metiera nada. Lo segundo ideal, que todo lo que se pueda extraer automáticamente, lo haga. ¿Yo os digo que tecleéis el número de milestones o lo pido al API de GitHub?